spec/hydra/pcdm/models/object_spec.rb

```diff
@@ -14,6 +14,45 @@
     end
   end
 
+  context 'when aggregated by other objects' do
+
+    before(:all) do
+      @collection1 = Hydra::PCDM::Collection.create
+      @collection2 = Hydra::PCDM::Collection.create
+      @parent_object = Hydra::PCDM::Object.create
+      @object =  Hydra::PCDM::Object.create
+      @collection1.members << @object
+      @collection1.save
+      @collection1.reload
+      @collection2.members << @object
+      @collection2.save
+      @collection2.reload
+      @parent_object.members << @object
+      @parent_object.save
+      @parent_object.reload
```

Comment:
There's a *lot* of `#reload` in the code you added to this PR, both in the setup and in the tests below. Are all of these needed?